.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -2,3 +2,19 @@
 **Asana Ticket:** [TICKET_NAME](TICKET_LINK)
 
 [Please include a brief description of what was changed]
+
+---
+
+Before getting review, please check the following:
+
+* [ ] Does frontend functionality render and work correctly in IE?
+* [ ] Has every endpoint that will be polled from the frontend been load-tested?
+* [ ] Do form elements have ARIA labels?
+* [ ] Have you checked for tech debt you can address in the area you're working in?
+* [ ] Have you considered the following cases that often require special handling:
+  * [ ] Green Line
+  * [ ] Foxboro CR
+  * [ ] Silver Line
+  * [ ] Unidirectional bus routes
+  * [ ] Ferry
+  * [ ] Red Line on the branches
```

Comment:
What kind of special handling does the Red Line require? I didn't think it was too different from most other routes; at least, I don't remember personally having to add special handling for it.

Reply:
Hmmmm, not sure what I had in mind here, I'll take the RL out unless I can think of a specific case that broke on it.